Ginger/GingerCore/ALM/RQM/ImportFromRQM.cs

```diff
@@ -840,7 +840,11 @@ private static void SaveItemFields(ObservableList<ExternalItemFieldBase> refresh
         public static ObservableList<ExternalItemFieldBase> GetOnlineItemFields(BackgroundWorker bw)
         {
             ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
-
+            if(bw == null)
+            {
+                bw = new BackgroundWorker();
+                bw.WorkerReportsProgress = true;
+            }
             //TODO : receive as parameters:
```

Comment:
in which cases the BackgroundWorker comes null?

Reply:
Export BF & AG and Items field configuration.